telemeta/models/responsability.py

```diff
@@ -0,0 +1,76 @@
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors: Anas Ghrab <anas.ghrab@gmail.com>
+
+from __future__ import division
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from telemeta.models.item import *
+from telemeta.models.collection import *
+
+class Authority(models.Model):
+    "Describe an autority"
+
+    first_name            = models.CharField(_('firstname'), max_length=50, help_text=_('First name in original language'))
+    last_name             = models.CharField(_('lastname'), max_length=50, help_text=_('Last name in original language'))
+    first_name_lat        = models.CharField(_('firstname_lat'), max_length=50, null=True, blank=True, help_text=_('First name in latin characters, usage form'))
+    last_name_lat         = models.CharField(_('lastname_lat'), max_length=50, null=True, blank=True, help_text=_('Last name in latin character, usage form'))
+    first_name_translit   = models.CharField(_('firstname_translit'), max_length=50, null=True, blank=True, help_text=_('First name transliterated'))
+    last_name_translit    = models.CharField(_('lastname_translit'), max_length=50, null=True, blank=True, help_text=_('Last name transliterated'))
+    birth                 = models.DateField(_('birth'), null=True, blank=True, help_text=_('YYYY-MM-DD'))
+    death                 = models.DateField(_('death'), null=True, blank=True, help_text=_('YYYY-MM-DD'))
+    biography             = models.TextField(_('biography'), null=True, blank=True)
+    isni                  = models.PositiveIntegerField(_('isni'), null=True, blank=True, help_text=_('International Standard Name Identifier'))
+    #photo                 =
+
+    class Meta:
+        db_table = 'media_authority'
+        verbose_name = _('authority')
+        verbose_name_plural = _('authorities')
+
+    def __str__(self):
+        return '%s %s' % (self.prenom, self.nom)
+
+class Role(models.Model):
+    "Describe a role related to items"
+
+    intitule            = models.CharField(_('role'), max_length=50, help_text=_('List of roles'))
+
+    class Meta:
+        db_table = 'media_role'
+        verbose_name = _('role')
+        verbose_name_plural = _('roles')
+
+    def __str__(self):
+        return '%s %s' % (self.intitule)
+
+class Responsability(models.Model):
+    "Describe a responsability in making an item"
+
+    authority            = models.ForeignKey('Authority', on_delete = models.CASCADE)
+    role                 = models.ForeignKey('Role', on_delete = models.CASCADE)
+    item                 = models.ForeignKey('MediaItem', on_delete = models.CASCADE)
+    collection           = models.ForeignKey('MediaCollection', on_delete = models.CASCADE)
```

Comment:
For those 4 fields, to avoid cascade deleting, I strongly advise to set `on_delete=models.SET_NULL`.
If not, in this case and for example, a Responsability could be deleted when the related Collection is deleted.